scripts/db_migrator.py

```diff
@@ -850,9 +850,21 @@ def version_3_0_6(self):
     def version_4_0_0(self):
         """
         Version 4_0_0.
-        This is the latest version for master branch
         """
         log.log_info('Handling version_4_0_0')
+        # Update state-db fast-reboot entry to enable if set to enable fast-reboot finalizer when using upgrade with fast-reboot
+        fastreboot_state = self.stateDB.get(self.stateDB.STATE_DB, 'FAST_REBOOT|system', '1')
+        if fastreboot_state == 'true':
```

Comment:
Do you not want to set the entry to `disable` when the fastreboot_state is not true?